packages/component/src/BasicTranscript.js

```diff
@@ -382,8 +385,33 @@ const InternalTranscript = ({ activityElementsRef, className }) => {
     [activityElementsRef, rootElementRef, scrollToBottomScrollTo]
   );
 
+  const scrollRelative = useCallback(
+    direction => {
+      const { current: rootElement } = rootElementRef;
+
+      if (!rootElement) {
+        return;
+      }
+
+      const scrollable = rootElement.querySelector('.webchat__basic-transcript__scrollable');
+
+      scrollTo(
+        {
+          scrollTop:
+            scrollable.scrollTop +
+            (direction === 'down' ? SCROLL_RELATIVE_DELTA_IN_PIXEL : -SCROLL_RELATIVE_DELTA_IN_PIXEL)
+        },
+        { behavior: 'smooth' }
+      );
+    },
+    [rootElementRef, scrollTo]
+  );
+
+  // Since there could be multiple instances of <BasicTranscript> inside the <Composer>, when the developer call `scrollXXX`, we need to call it on all instances.
```

Comment:
```suggestion
  // Since there could be multiple instances of <BasicTranscript> inside the <Composer>, when the developer calls `scrollXXX`, we need to call it on all instances.
```